src/ConvertCommand.php

```diff
@@ -55,6 +55,7 @@ protected function configure()
             ->setDescription('Convert files')
             ->addOption('exclude', 'e', InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Directories to exclude', ['vendor'])
             ->addOption('dry-run', null, InputOption::VALUE_NONE, 'Displays diff instead of modifying files')
+            ->addOption('nullable-types', null, InputOption::VALUE_NONE, 'Uses nullable types (PHP 7.1+) instead of `null` default value')
```

Comment:
Would a catch-all option to enable all PHP 7.1 features (e.g. `iterable` pseudo-type) be preferred?

Reply:
I would enable this feature by default. Why not providing a flag to disable it but I think we should generate the best code possible by default.

Reply:
Sure, now it is enabled by default.